BNomad/View/ProfileView/ProfileGraphCell.swift

```diff
@@ -12,6 +12,7 @@ class ProfileGraphCell: UICollectionViewCell {
     
     //MARK: -Properties
     static let identifier = "ProfileGraphCell"
+    static var addedWeek:Int = 0
```

Comment:
```suggestion
    static var addedWeek: Int = 0
```
공백 파일 한번 전체적으로 다시 봐주세욧👍

Reply:
넵